symengine/lib/symengine_wrapper.pyx

```diff
@@ -462,7 +463,6 @@ cdef class Basic(object):
         return Basic._richcmp_(A, B, op)
 
     def _richcmp_(Basic A, Basic B, int op):
-        from sympy import Rel
```

Comment:
Do you only use `==` and `!=`? (For `_coeff_isneg`, see #114)
If so, we can move the import further down, so you won't be affected.
Reason is that @certik and I wanted to remove these implicit conversions to sympy as it maybe a silent bottleneck for some users.